SFDigitalServices/sf-dahlia-web
# DAH-2858 feat: sort fcfs sales and render below open listings

## Description

- When displaying listings on the sales directory, check if a listing is a FCFS Sales listing, and then sort that listing by descending application start date
- The FCFS Sales listings should be behind the FCFS feature flag.

NOTE: Reviewer should test both tickets when reviewing this PR.

## Jira ticket

https://sfgovdt.jira.com/browse/DAH-2858
https://sfgovdt.jira.com/browse/DAH-2910

## Checklist before requesting review

### Version Control


### Code quality


### Review instructions


### Request review


## Review Instructions
- Go to Sales directory
- Ensure FCFS Sales listings show up under the open listings (ORA Should be the first one)
- Ensure that FCFS Sales listings are in the correct order (newest Start Date first)
- If you want, try updating the Start Date of a FCFS Sales listing in Salesforce and ensure the changes are reflected in the displayed sort order
